.gitignore

```diff
@@ -1 +1,2 @@
+.DS_store
```

Comment:
I usually like to have it set in the project's gitignore so that people don't accidentally end up committing it when they don't have it set globally, but either is fine with me :).